### PR TITLE
FE-Personal-data

### PR DESCRIPTION
**Pull request by akenaaa**
image

### CODE REVIEW

File: client/src/base/colors.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ export const colors = {
   white: '#FFFFFF',
   black: '#000000',
   cardBg: '#737399',
+  personalDataCardBg: '#E4F3FB',
```
Comment (eyesofdaveed) [inline client/src/base/colors.js]: use more generic name

File: client/src/organism/PersonalData.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Flexbox } from '@atoms';
+import { sizes } from '@base/index';
+import { colors } from '@base';
```
Comment (eyesofdaveed) [inline client/src/organism/PersonalData.jsx]: import { sizes, colors} from '@base'

File: client/src/organism/PersonalData.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+export const PersonalDataCard = styled.div`
+  background-color: ${colors.personalDataCardBg};
+  border-radius: 30px;
+  padding: 2%;
+  display: flex;
+  width: fit-content;
+
+  
+`;
```
Comment (eyesofdaveed) [inline client/src/organism/PersonalData.jsx]: whitespaces

File: client/src/organism/PersonalData.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+export const PersonalDataCardText = styled.div`
+  background-color: ${colors.white};
+  border-radius: 18px;
+  padding: 10px;
+  width: 300px;
+  // display: flex;
+  // justify-content: space-between;
+`;
+
+export const TextOfPerson = styled.p`
```
Comment (eyesofdaveed) [inline client/src/organism/PersonalData.jsx]: `styled(Text)`

File: client/src/organism/PersonalData.jsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  // white-space: nowrap;
+  
+`;
+
+export const PersonalData = () => {
+  return (
+    <PersonalDataCard>
+      <Flexbox direction="column" gap="22px" align="start">
+        <Flexbox justify="space-between" width="100%">
+          <TextOfPerson fontSize={sizes.medium}>ИИН</TextOfPerson>
```
Comment (eyesofdaveed) [inline client/src/organism/PersonalData.jsx]: Label

File: client/src/organism/PersonalData.jsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+          <PersonalDataCardText>
+            <TextOfPerson fontSize={sizes.medium}>
```
Comment (eyesofdaveed) [inline client/src/organism/PersonalData.jsx]: Value

**Review by eyesofdaveed — APPROVED** [review]

File: client/src/organism/PersonalData.jsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  width: 300px;
+`;
+
+export const PersonalData = () => {
+  return (
+    <Card padding="30px" display="flex" width="fit-content" bgColor={colors.cardMainBg} >
+      <Flexbox direction="column" gap="22px" align="start">
+        <Flexbox justify="space-between" width="100%">
+          <Text color= 'black'fontSize={sizes.medium} >ИИН</Text>
+          <ValueBackround>
```
Comment (Bashkanitto) [inline client/src/organism/PersonalData.jsx]: Backround or back'G'round?

**Comment by eyesofdaveed** [comment]
@akenaaa Please fix the mc and merge it to the main

**Review by Bashkanitto — APPROVED** [review]

**Review by Bashkanitto — APPROVED** [review]

**Pull request merged**